react/features/reactions-muted/constants.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
i don't think there's need for a separate `reactions-muted` structure, since is just a part of the reactions features